analyzer/throttle.h

```diff
@@ -30,32 +30,16 @@
  */
 #define SUSCAN_THROTTLE_RESET_THRESHOLD 1000000000ll
```

Comment:
Add `_NS` suffix. Since multiple time units are used across the code, it is important to specify them.

Reply:
That one is yours ;)

I'll add a commit to make it consistent :)

Reply:
Actually, this is *not* a time constant.  It is used for sample count.

Reply:
My bad :_D I'm fixing it myself, it's obvious it was not clear enough to begin with.